src/components/PhoneNumberInput/style.scss

```diff
@@ -64,7 +64,7 @@
   }
 
   .rrui__select__options:not(.rrui__select__options--menu) {
-    width: calc(100% + 2.4 * $unit-rrui);
+    width: calc(100% + #{2.4 * $unit-rrui});
```

Comment:
Why `#{2.4 * $unit-rrui}` and not `2.4 *  #{$unit-rrui}`?

Reply:
Not sure. But the contributor was probably just following the pattern in the earlier lines of Sass before it.